common/cah-model.cpp

```diff
@@ -0,0 +1,196 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2020 Intel Corporation. All Rights Reserved.
+
+#include <algorithm>
+#include "cah-model.h"
+#define GLFW_INCLUDE_NONE
+#include "fw-update-helper.h"
+#include "model-views.h"
+
+using namespace rs2;
+
+// This registration should happen once in the SW runtime life.
+static bool registered_to_callback = false;
+
+// This variable is global for protecting the case when the callback will be called when the device model no longer exist.
+static std::atomic<rs2_calibration_status> calib_status = RS2_CALIBRATION_RETRY; 
```

Comment:
Can we denote that it's global somehow? "global_calibration_status" or something similar?

Reply:
Sure, will be updated